tensorboard/webapp/hparams/_types.ts

```diff
@@ -13,12 +13,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 import {
-  DiscreteHparamValues,
-  DomainType,
   HparamSpec,
   MetricSpec,
 } from '../runs/data_source/runs_data_source_types';
 
+export {DiscreteFilter, IntervalFilter} from '../widgets/data_table/types';
```

Comment:
I don't like re-exporting things but there are several files which import and then reexoport FROM this file and I would rather wait to cleaning the whole mess up until AFTER launching HParams in Timeseries Phase 1.

Reply:
I added it to our spreadsheet https://docs.google.com/spreadsheets/d/1RBodm1hj3-njiowUeRXQe3h1GlVb70Ce1JvZw_Cp_Yc/edit?resourcekey=0-ehUesPxhJhOONsW6w_SXsA#gid=0